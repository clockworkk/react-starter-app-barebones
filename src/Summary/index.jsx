import React from 'react';

import './styles.scss';

const Summary = () => (
  <div className="summary">
    Content.
  </div>
);

export default Summary;
