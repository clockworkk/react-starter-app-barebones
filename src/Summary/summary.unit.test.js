/* eslint-disable */

import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Summary from './';

Enzyme.configure({ adapter: new Adapter() });

describe('src::Summary', () => {

  describe('rendering', () => {
    
    const component = renderer.create(
      <Summary />
    );

    test('renders properly and matches snapshot.', () => {
      let tree = component.toJSON();
      jestExpect(tree).toMatchSnapshot();
    });
  });
});
