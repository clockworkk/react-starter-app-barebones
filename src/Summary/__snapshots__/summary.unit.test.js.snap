// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src::Summary rendering renders properly and matches snapshot. 1`] = `
<div
  className="summary"
>
  Content.
</div>
`;
